packages/next/src/client/components/router-reducer/apply-router-state-patch-to-tree.ts

```diff
@@ -2,6 +2,7 @@ import type {
   FlightRouterState,
   FlightSegmentPath,
 } from '../../../server/app-render/types'
+import { DEFAULT_SEGMENT_KEY } from '../../../shared/lib/constants'
```

Comment:
@ztanner Seems importing these constants resulting in client bundle size increasing @ztanner , shall we have a client constants for `client/components` files? I think `shared/lib/constants` was not included in client chunk